[BUG] Compilation fails

```
[ 65%] Building CXX object CMakeFiles/hyprlock.dir/src/renderer/widgets/Background.cpp.o
/home/alba4k/.cache/paru/clone/hyprlock-git/src/hyprlock/src/core/hyprlock.cpp: In member function ‘void CHyprlock::onKey(uint32_t, bool)’:
/home/alba4k/.cache/paru/clone/hyprlock-git/src/hyprlock/src/core/hyprlock.cpp:661:57: error: ‘class CHyprlock’ has no member named ‘m_tGraceEnd’; did you mean ‘m_tGraceEnds’?
  661 |     if (std::chrono::system_clock::now() < g_pHyprlock->m_tGraceEnd) {
      |                                                         ^~~~~~~~~~~
      |                                                         m_tGraceEnds
/home/alba4k/.cache/paru/clone/hyprlock-git/src/hyprlock/src/core/hyprlock.cpp: In member function ‘std::string CHyprlock::spawnSync(const std::string&)’:
/home/alba4k/.cache/paru/clone/hyprlock-git/src/hyprlock/src/core/hyprlock.cpp:843:50: warning: ignoring attributes on template argument ‘int (*)(FILE*)’ [-Wignored-attributes]
  843 |     const std::unique_ptr<FILE, decltype(&pclose)> pipe(popen(cmd.c_str(), "r"), pclose);
      |                                                  ^
[ 68%] Building CXX object CMakeFiles/hyprlock.dir/src/renderer/widgets/IWidget.cpp.o
[ 71%] Building CXX object CMakeFiles/hyprlock.dir/src/renderer/widgets/Label.cpp.o
/home/alba4k/.cache/paru/clone/hyprlock-git/src/hyprlock/src/core/hyprlock.cpp: In member function ‘void CHyprlock::spawnAsync(const std::string&)’:
/home/alba4k/.cache/paru/clone/hyprlock-git/src/hyprlock/src/core/hyprlock.cpp:819:14: warning: ignoring return value of ‘ssize_t write(int, const void*, size_t)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
  819 |         write(socket[1], &grandchild, sizeof(grandchild));
      |         ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/alba4k/.cache/paru/clone/hyprlock-git/src/hyprlock/src/core/hyprlock.cpp:827:9: warning: ignoring return value of ‘ssize_t read(int, void*, size_t)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
  827 |     read(socket[0], &grandchild, sizeof(grandchild));
      |     ~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
make[3]: *** [CMakeFiles/hyprlock.dir/build.make:174: CMakeFiles/hyprlock.dir/src/core/hyprlock.cpp.o] Error 1
make[3]: *** Attesa per i processi non terminati....
make[2]: *** [CMakeFiles/Makefile2:83: CMakeFiles/hyprlock.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:90: CMakeFiles/hyprlock.dir/rule] Error 2
make: *** [Makefile:170: hyprlock] Error 2
```

--- a/src/core/hyprlock.cpp
+++ b/src/core/hyprlock.cpp
@@ -658,7 +658,7 @@ std::optional<std::string> CHyprlock::passwordLastFailReason() {
 void CHyprlock::onKey(uint32_t key, bool down) {
     const auto SYM = xkb_state_key_get_one_sym(m_pXKBState, key + 8);
 
-    if (std::chrono::system_clock::now() < g_pHyprlock->m_tGraceEnd) {
+    if (down && std::chrono::system_clock::now() < g_pHyprlock->m_tGraceEnds) {
         unlockSession();
         return;
     }

